awx_collection/tests/integration/targets/tower_organization/tasks/main.yml

```diff
@@ -80,31 +80,19 @@
       - "result is not changed"
 
 # Test behaviour common to all tower modules
-- name: Check that SSL is available
-  tower_organization:
-    name: Default
-  register: result
-
-- assert:
-    that: result is not changed
-
 - name: Check that SSL is available and verify_ssl is enabled (task must fail)
   tower_organization:
     name: Default
     validate_certs: true
-  environment:
-    TOWER_CERTIFICATE: /dev/null  # force check failure
   ignore_errors: true
   register: check_ssl_is_used
 
 - name: Check that connection failed
```

Comment:
This is gonna fail if we run against a tower with good ssl certs, both in your change and before you change. It would be nice to have a test that actually worked all the time